quinn-proto/src/endpoint.rs

```diff
@@ -482,6 +512,10 @@ impl Endpoint {
             }
         };
 
+        let incoming_idx = self.incoming_buffers.insert(Default::default());
```

Comment:
Please avoid `Default::default()` in favor of using the type name.

Reply:
Done.